kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/taskassigning/UserType.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.server.api.model.taskassigning;
+
+public class UserType {
```

Comment:
Could be an enum. Btw, isn't this already available somewhere in jBPM API?

Reply:
I've looked, but no definition exists in jBPM API. Constants where upper-cased.